src/test/java/org/broadinstitute/hellbender/engine/GATKToolUnitTest.java

```diff
@@ -1,12 +1,8 @@
 package org.broadinstitute.hellbender.engine;
 
+import com.sun.istack.Nullable;
```

Comment:
For consistency with the rest of the source, please use javax.annotation.Nullable

Reply:
ok